pkg/webui/components/sidebar/side-menu/item/link.js

```diff
@@ -43,6 +44,13 @@ const MenuLink = ({ icon, title, path, onClick, exact, disabled }) => {
     <NavLink to={path} className={className} end={exact} onClick={onClick}>
       {icon && <Icon icon={icon} className={classnames(style.icon)} />}{' '}
       {!isMinimized && <Message content={title} />}
+      {isMinimized && (
```

Comment:
The regular items should also show the flyout on hover, when the sidebar is collapsed.